### PR TITLE
refactored readme

### PR DESCRIPTION
**Pull request by iamrajiv**
### Summary

Fixed grammatical mistakes and formatted README.md

### CODE REVIEW

**Comment by iamrajiv** [comment]
@guanlan Please review.

**Comment by iamrajiv** [comment]
@javierguerragiraldez @locao Please review.

**Comment by iamrajiv** [comment]
@hutchic Is it ready to be merged?

File: README.md [hunk 1]
```diff
@@ -40,29 +40,24 @@ transformations, and more through plugins.
 
 Kong has been built with the following leading principles:
 
-* **High Performance**: Sub-millisecond processing latency to support mission 
+- **High Performance**: Sub-millisecond processing latency to support the mission
   critical use cases and high throughput.
```
Comment (javierguerragiraldez) [inline README.md:44]: i think this "the" isn't appropriate here.

File: README.md [hunk 2]
```diff
@@ -40,29 +40,24 @@ transformations, and more through plugins.
...
-- **Kubernetes-Native**: Declaratively configure Kong with native Kubernetes CRDs
-  using the official Ingress Controller to route and connect all L4 + L7 traffic. 
-- **Dynamic Load Balancing**: Load balance traffic across multiple upstream
-  services.
-- **Hash-based Load Balancing**: Load balance with consistent hashing/sticky
-  sessions.
+- **Cloud-Native**: Platform agnostic, Kong can run on any platform - from bare metal to containers - and it can run on every cloud natively.
+- **Kubernetes-Native**: Declaratively configure Kong with native Kubernetes CRDs using the official Ingress Controller to route and connect all L4 + L7 traffic.
+- **Dynamic Load Balancing**: Load balance traffic across multiple upstream services.
+- **Hash-based Load Balancing**: Load balance with consistent hashing/sticky sessions.
```
Comment (javierguerragiraldez) [inline README.md:55]: this only removes wrapping in the sources, no effect on the result.

File: README.md [hunk 3]
```diff
@@ -114,25 +103,24 @@ code, other repos are also under active development:
   for Kong.
 - [Kong CloudFormation](https://github.com/Kong/kong-dist-cloudformation):
   Kong in a 1-click deployment for AWS EC2.
-- [Kong AWS AMI](https://aws.amazon.com/marketplace/pp/B06WP4TNKL): Kong AMI on
-  the AWS Marketplace.
+- [Kong AWS AMI](https://aws.amazon.com/marketplace/pp/B06WP4TNKL): Kong AMI on the AWS Marketplace.
 - [Kong on Microsoft Azure](https://github.com/Kong/kong-dist-azure): Run Kong
-  using Azure Resource Manager.
+  using the Azure Resource Manager.
```
Comment (javierguerragiraldez) [inline README.md:108]: i think "the" isn't appropriate here.  how is it on microsoft docs?

File: README.md [hunk 4]
```diff
@@ -114,25 +103,24 @@ code, other repos are also under active development:
...
 - [Kong on Heroku](https://github.com/heroku/heroku-kong): Deploy Kong on
   Heroku in one click.
 - [Kong and Instaclustr](https://www.instaclustr.com/solutions/managed-cassandra-for-kong/): Let
-  Instaclustr manage your Cassandra cluster.
+  Instaclustr manages your Cassandra cluster.
```
Comment (javierguerragiraldez) [inline README.md:112]: typo?

File: README.md [hunk 5]
```diff
@@ -114,25 +103,24 @@ code, other repos are also under active development:
...
 - [Nightly Builds][kong-nightly-master]: Builds of the master branch available
-  every morning at about 9AM PST.
+  every morning at about 9 AM PST.
 
 You can find every supported distribution at the [official installation page](https://konghq.com/install/).
 
 ## Development
 
-If you are planning on developing on Kong, you'll need a development
+If you are planning on developing in Kong, you'll need a development
```
Comment (javierguerragiraldez) [inline README.md:120]: personally, i use "developing in" a language, but "developing on" a platform.  not sure if there's any grammar rule relevant.

File: README.md [hunk 6]
```diff
@@ -114,25 +103,24 @@ code, other repos are also under active development:
...
 installation. The `next` branch holds the latest unreleased source code.
 
-You can read more about writing your own plugins in the [Plugin Development
+You can read more about writing your plugins in the [Plugin Development
```
Comment (javierguerragiraldez) [inline README.md:123]: i feel that "your plugins" makes sense if they've been mentioned before.  the first time "your own plugins" sounds better.

File: README.md [hunk 7]
```diff
@@ -189,9 +177,9 @@ $ make dev
 
 Kong relies on three test suites using the [busted] testing library:
 
-* Unit tests
-* Integration tests, which require Postgres and Cassandra to be up and running
-* Plugins tests, which require Postgres to be running
+- Unit tests
+- Integration tests, which require Postgres and Cassandra to be up and running
+- Plugins tests, which require Postgres to be running
```
Comment (javierguerragiraldez) [inline README.md:182]: AFAICT, most markup engines treat '*' and '-' as equivalent when defining lists.

File: README.md [hunk 8]
```diff
@@ -238,15 +226,15 @@ $ make lint
...
-| `test-all`         | Run all unit + integration + plugins tests at once     |
+|               Name | Description                                        |
+| -----------------: | -------------------------------------------------- |
+|          `install` | Install the Kong luarock globally                  |
+|              `dev` | Install development dependencies                   |
+|             `lint` | Lint Lua files in `kong/` and `spec/`              |
+|             `test` | Run the unit tests suite                           |
+| `test-integration` | Run the integration tests suite                    |
+|     `test-plugins` | Run the plugins test suite                         |
+|         `test-all` | Run all unit + integration + plugins tests at once |
```
Comment (javierguerragiraldez) [inline README.md:237]: not sure if it has any effect or not.  if not, it only makes it harder to maintain.

**Review by javierguerragiraldez — CHANGES_REQUESTED** [review]

**Comment by iamrajiv** [comment]
@javierguerragiraldez Take a look now. I have formatted the README using Prettier and Grammarly.

**Comment by iamrajiv** [comment]
@javierguerragiraldez Take a look now. I have formatted the README using Prettier and Grammarly.

**Pull request closed without merging**